basic-algorithms.js

```diff
@@ -1,7 +1,44 @@
-// Names and Input
+var hacker1 = "alonso"
+
+console.log("The driver/'s name is XXXX")
+
+var hacker2 = prompt("Please type in the navigator\'s name: ");
+
+console.log("The navigator\'s name is YYYY");
+
+var hacker1Length = hacker1.length;
+
+var hacker2length = hacker2.length;
+
+if (hacker1Length > hacker2length) {
```

Comment:
you don't have to store these as variables, its common practise to just pass these straight in :)